common/src/main/java/gov/cms/ab2d/common/model/JobOutput.java

```diff
@@ -39,4 +39,10 @@
 
     @NotNull
     private Boolean downloaded = false;
+
+    @NotNull
```

Comment:
Why not just store the checksum in hex right away?

Reply:
I was looking at what DPC did in their app, they kept it in the byte[] until displaying it to the user, I wasn't sure if there was a reason they were waiting to convert to hex.